src/Widgets/WindowClone.vala

```diff
@@ -28,7 +28,7 @@ namespace Gala {
 
         public override ActorBox get_bounding_box () {
             var scale_factor = InternalUtils.get_ui_scaling_factor ();
-            var size = shadow_size * scale_factor;
+            var size = shadow_size + shadow_spread * scale_factor;
```

Comment:
Haven't tested this, but wouldn't this make more sense as

```suggestion
            var size = (shadow_size + shadow_spread) * scale_factor;
```

Reply:
Yeah you're probably right!